pkgs/by-name/ri/ride/package.nix

```diff
@@ -0,0 +1,130 @@
+{ lib
+, stdenv
```

Comment:
Do you think we could use stdenvNoCC ?

Reply:
I guess it could be done, though I am not sure if it decreases the closure size (It's only used for platform info queries)